diff -uNrp a/ext/meson.build b/ext/meson.build
--- a/ext/meson.build	2020-08-20 16:14:26.000000000 +0100
+++ b/ext/meson.build	2020-08-23 14:38:26.171082319 +0100
@@ -1,69 +1,10 @@
-subdir('assrender')
-subdir('aom')
-subdir('avtp')
-subdir('bs2b')
-subdir('bz2')
-subdir('chromaprint')
-subdir('closedcaption')
-subdir('colormanagement')
-subdir('curl')
-subdir('dash')
-subdir('dc1394')
-subdir('directfb')
-subdir('dtls')
 subdir('dts')
-subdir('faac')
 subdir('faad')
-subdir('fdkaac')
-subdir('flite')
-subdir('fluidsynth')
-subdir('gme')
-subdir('gsm')
-subdir('hls')
-subdir('iqa')
-subdir('kate')
-subdir('ladspa')
 subdir('libde265')
 subdir('libmms')
-subdir('lv2')
-subdir('mdns')
-subdir('modplug')
 subdir('mpeg2enc')
 subdir('mplex')
-subdir('musepack')
-subdir('neon')
-subdir('ofa')
-subdir('openal')
-subdir('opencv')
-subdir('openexr')
-subdir('openh264')
-subdir('openjpeg')
-subdir('openmpt')
-subdir('openni2')
-subdir('opus')
-subdir('resindvd')
-subdir('rsvg')
 subdir('rtmp')
-subdir('sbc')
-subdir('sctp')
-subdir('smoothstreaming')
-subdir('sndfile')
-subdir('soundtouch')
-subdir('spandsp')
-subdir('srt')
-subdir('srtp')
 subdir('svthevcenc')
-subdir('teletextdec')
-subdir('ttml')
-subdir('voaacenc')
 subdir('voamrwbenc')
-subdir('vulkan')
-subdir('wayland')
-subdir('webrtc')
-subdir('webrtcdsp')
-subdir('webp')
-subdir('wildmidi')
-subdir('wpe')
 subdir('x265')
-subdir('zxing')
-subdir('zbar')

diff -uNrp a/gst/meson.build b/gst/meson.build
--- a/gst/meson.build	2020-08-20 16:14:27.000000000 +0100
+++ b/gst/meson.build	2020-08-23 14:38:26.172082317 +0100
@@ -1,18 +1,4 @@
-foreach plugin : ['accurip', 'adpcmdec', 'adpcmenc', 'aiff', 'asfmux',
-                  'audiobuffersplit', 'audiofxbad', 'audiomixmatrix',
-                  'audiolatency', 'audiovisualizers', 'autoconvert', 'bayer',
-                  'camerabin2', 'coloreffects', 'debugutils', 'dvbsubenc',
-                  'dvbsuboverlay', 'dvdspu', 'faceoverlay', 'festival',
-                  'fieldanalysis', 'freeverb', 'frei0r', 'gaudieffects', 'gdp',
-                  'geometrictransform', 'id3tag', 'inter', 'interlace',
-                  'ivfparse', 'ivtc', 'jp2kdecimator', 'jpegformat', 'librfb',
-                  'midi', 'mpegdemux', 'mpegpsmux', 'mpegtsdemux', 'mpegtsmux',
-                  'mxf', 'netsim', 'onvif', 'pcapparse', 'pnm', 'proxy',
-                  'rawparse', 'removesilence', 'rist', 'rtmp2', 'rtp', 'sdp',
-                  'segmentclip', 'siren', 'smooth', 'speed', 'subenc', 'switchbin',
-                  'timecode', 'transcode', 'videofilters',
-                  'videoframe_audiolevel', 'videoparsers', 'videosignal',
-                  'vmnc', 'y4m']
+foreach plugin : ['dvbsuboverlay', 'dvdspu', 'siren']
   if not get_option(plugin).disabled()
     subdir(plugin)
   endif
diff -uNrp a/meson.build b/meson.build
--- a/meson.build	2020-08-20 16:14:27.000000000 +0100
+++ b/meson.build	2020-08-23 14:38:26.173082315 +0100
@@ -279,6 +279,8 @@
     fallback : ['gst-plugins-base', 'app_dep'])
 gstaudio_dep = dependency('gstreamer-audio-1.0', version : gst_req,
     fallback : ['gst-plugins-base', 'audio_dep'])
+gstcodecparsers_dep = dependency('gstreamer-codecparsers-1.0', version : gst_req,
+    fallback : ['gst-plugins-bad', 'codecparsers_dep'])
 gstfft_dep = dependency('gstreamer-fft-1.0', version : gst_req,
     fallback : ['gst-plugins-base', 'fft_dep'])
 gstriff_dep = dependency('gstreamer-riff-1.0', version : gst_req,
@@ -461,14 +463,9 @@
   plugins_pkgconfig_install_dir = disabler()
 endif
 
-subdir('gst-libs')
 subdir('gst')
-subdir('sys')
 subdir('ext')
-subdir('tests')
 subdir('data')
-subdir('tools')
-subdir('pkgconfig')
 
 if have_orcc
   update_orc_dist_files = find_program('scripts/update-orc-dist-files.py')

